tombo/tombo_plot_genome_locations.xml

```diff
@@ -0,0 +1,78 @@
+<tool id="tombo_plot_genome_locations" name="Tombo plot genome locations" version="@VERSION@">
+    <description>Plotting amperage signal from nanopore reads at the locations specified</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+    mkdir -p ./fast5/ &&
+    mkdir -p ./fast5s_control/ &&
+
+    #if $raw_signals.ext == 'fast5.tar':
+        tar -xf '$raw_signals' -C ./fast5/ &&
+    #elif $raw_signals.ext == 'fast5.tar.gz':
+        tar -xzf '$raw_signals' -C ./fast5/ &&
+    #elif $raw_signals.ext == 'fast5.tar.bz2':
+        tar -xjf '$raw_signals' -C ./fast5/ &&
+    #end if
+
+    #if $include_control.include_control_bool:
+        #if $control_signals.ext == 'fast5.tar':
+            tar -xf '$control_signals' -C ./fast5s_control/ &&
+        #elif $control_signals.ext == 'fast5.tar.gz':
+            tar -xzf '$control_signals' -C ./fast5s_control/ &&
+        #elif $control_signals.ext == 'fast5.tar.bz2':
+            tar -xjf '$control_signals' -C ./fast5s_control/ &&
+        #end if
+    #end if
+
+    tombo plot most_significant --fast5-basedir ./fast5/
+        #if $include_control.include_control_bool:
+            --control-fast5-basedirs ./fast5s_control/ 
+        #end if
+        --genome-locations '$genome_locations'
+        --pdf-filename '$output' 
+        '$model_type'
+        --overplot-threshold '$overplot_threshold'
+        --overplot-type '$overplot_type'
+        --num-bases '$num_bases'
+        --corrected-group '$corrected_group' 
+
+    ]]></command>
+    <inputs>
+        <expand macro="raw_signals" />
+        <conditional name="include_control">
+            <param name="include_control_bool" type="boolean" label="Plot control samples? (samples with only canonical nucleotides)"/>
+            <when value="true">
```

Comment:
According to our best practise guideline, booleans should not be used for conditionals: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html?highlight=boolean#booleans

Reply:
Good to know. I'll fix this in my next update to it.